Content.Server/ReagentOnItem/EntitySystems/SpaceLubeOnItemSystem.cs

```diff
@@ -0,0 +1,62 @@
+using Content.Shared.IdentityManagement;
+using Content.Shared.Popups;
+using Content.Shared.ReagentOnItem;
+using Content.Shared.Throwing;
+using Robust.Shared.Containers;
+using Robust.Shared.Random;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Inventory;
+using Content.Shared.Examine;
+
+namespace Content.Server.ReagentOnItem;
+
+public sealed class SpaceLubeOnItemSystem : EntitySystem
+{
+    [Dependency] private readonly ThrowingSystem _throwing = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] private readonly InventorySystem _inventory = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ContainerGettingInsertedAttemptEvent>(OnHandPickUp);
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ExaminedEvent>(OnExamine);
+    }
+
+    private void OnHandPickUp(EntityUid uid, SpaceLubeOnItemComponent component, ContainerGettingInsertedAttemptEvent args)
+    {
+        _inventory.TryGetSlotEntity(args.Container.Owner, "gloves", out var gloves);
+
+        if (HasComp<NonStickSurfaceComponent>(gloves))
+            return;
```

Comment:
The TryGetSlotEntity needs to be part of the if. Some entities don't have a gloves slot like monkeys.